index.html

```diff
@@ -241,20 +374,23 @@ <h3 class="skills">Terminal</h3>
             </article>
         </section>
         <section class="form-section">
+            <h2 class="form-title">
+                I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.
+            </h2>
             <article class="form-container">
-                <h2 class="form-title">
-                    I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.
-                </h2>
-                <form action="https://formspree.io/f/xyyoqkjw" method="post" class="contact-form">
-                    <input type="text" name="full-name" maxlength="30" placeholder="Full Name" required>
+                
+                <form class="contact-form" action="https://formspree.io/f/xyyoqkjw" method="post" >
+                    <input type="text" name="First-Name" placeholder="First name" class="form-input" >
+                    <input type="text" name="full-name" maxlength="30" placeholder="Full Name" class="mbil" >
+                    <input class="hide" type="text" name="Last-name" maxlength="30" placeholder="Last Name" >
                     <input type="email" name="email" placeholder="Email address" required>
                     <textarea name="message"  cols="30" rows="10" placeholder="Enter text here" required maxlength="500"></textarea>
                     <button type="submit" class="btn-contact">Get in touch</button>
                 </form>
             </article>
```

Comment:
- [ ] Kindly fix your design to match the template by giving the appropriate width for input fields.

| Your design  | Figma design|
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/53402600/178015428-4381e362-95d9-4b6a-b33d-26f67743bf7c.png) | ![image](https://user-images.githubusercontent.com/53402600/178015504-90c322d4-f818-4624-afdb-d4b9b7f7dbc6.png) |